plugin/PDFView.qml

```diff
@@ -141,6 +143,11 @@ SilicaFlickable {
             returnY.from = pt.y - deltaY
             returnY.to = pt.y
             quickScrollAnimation.pageTo = pageId
+            if (focusItem !== undefined) {
+                quickScrollAnimation.focusItem = focusItem
```

Comment:
Just thinking, but since both animations only pass the property further to focusAnimation, could just always assign the directly and save some lines. Though now this function doesn't need to know animations trigger another. Either way ok.